src/AuthorizationServiceProvider.php

```diff
@@ -24,4 +27,22 @@ public function configureTipoffPackage(TipoffPackage $package): void
             ->name('authorization')
             ->hasConfigFile();
     }
+
+    public function registeringPackage()
+    {
+        Nova::booted(function () {
+            // Register serving function only after Nova is booted, this should
+            // ensure we are "last in line" and able to replace the default gate definition
+            Nova::serving(function () {
+                Gate::define('viewNova', function ($user) {
+                    if ($user instanceof UserInterface) {
+                        return app()->environment('testing') ||
+                            $user->hasPermissionTo('access admin');
+                    }
+
+                    return false;
+                });
+            });
+        });
```

Comment:
Getting this into the package makes me really happy! Thank you!

Reply:
I'll think through the consequences of it. Basically, we're overwriting the nova gate in the application.

Reply:
Yes -- but the problem is it only works if we're last and the gate we want to replace has already been defined.  If the order flips and we're not last any longer, then it doesn't work.